lemur/static/app/angular/app.js

```diff
@@ -1,139 +1,158 @@
 'use strict';
 
-var lemur = angular
-  .module('lemur', [
-    'ui.router',
-    'ngTable',
-    'ngAnimate',
-    'chart.js',
-    'restangular',
-    'angular-loading-bar',
-    'ui.bootstrap',
-    'angular-spinkit',
-    'toaster',
-    'uiSwitch',
-    'mgo-angular-wizard',
-    'satellizer',
-    'ngLetterAvatar',
-    'angular-clipboard',
-    'ngFileSaver'
-  ])
-  .config(function ($stateProvider, $urlRouterProvider, $authProvider, AuthenticationService) {
-    $urlRouterProvider.otherwise('/welcome');
+(function() {
+  var lemur = angular
+    .module('lemur', [
+      'ui.router',
+      'ngTable',
+      'ngAnimate',
+      'chart.js',
+      'restangular',
+      'angular-loading-bar',
+      'ui.bootstrap',
+      'angular-spinkit',
+      'toaster',
+      'uiSwitch',
+      'mgo-angular-wizard',
+      'satellizer',
+      'ngLetterAvatar',
+      'angular-clipboard',
+      'ngFileSaver'
+    ]);
+
+
+  function fetchData() {
+    var initInjector = angular.injector(['ng']);
+    var $http = initInjector.get('$http');
+
+    return $http.get('http://localhost:8000/api/1/auth/providers').then(function(response) {
```

Comment:
Oh right you are, thanks. Turns out my problem was that I ran "gulp build package" in a single command and this way it doesn't do it for some reason.